source/VideoPlayer.js

```diff
@@ -1873,7 +1877,7 @@
 		*/
 		timeUpdate: function(sender, e) {
 			//* Update _this.duration_ and _this.currentTime_
-			if (!e && e.srcElement) {
+			if (!e && e.srcElement || e.currentTime === undefined) {
```

Comment:
Ah yes, `e.currentTime == null` should evaluate to true when `e.currentTime` is either `undefined` or `null`